travis_configure.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+# Script for setting up OS specific CI test environment on travis-ci.org
+#
+# Copyright (C) 2017 Andreas Bok Andersen <andreas.bok@sociomantic.com>
+#
+
+set -e
+
+# Script which is run before installation script is called
+#
+# Installs fpm needed for deb packaging
+#
+# Returns 0
+before_install()
+{
+    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+        if [[ "$DIST_PACKAGE_TARGET" == "DEB" ]]; then
+            gem install fpm
+        fi
```

Comment:
**fpm** is only used to build the deb package. The **rpm** package is built on the docker image. 
Just to avoid installing unnecessary apps.